File: 2977-check-if-a-string-is-an-acronym-of-words/solution.py
class Solution:
    def isAcronym(self, words: List[str], s: str) -> bool:
        list1=[]
        for i in words:
            list1.append(i[0])
        print(list1)
        st=''
        for i in list1:
            st+=i
        return st==s

        

File: 2977-check-if-a-string-is-an-acronym-of-words/solution.java
class Solution {
    public boolean isAcronym(List<String> words, String s) {
        String st="";
        for(int i=0;i<words.size();i++){
            String str=words.get(i);
            char[] ch=str.toCharArray();
            st+=ch[0];
        }
        if(st.equals(s)){
            return true;
        }
        return false;
    }
}
